### PR TITLE
Astro 1198 Update Validation Documentation

### PR DESCRIPTION
**Pull request by amalyah**
https://rocketcom.atlassian.net/browse/ASTRO-1198
Builds off of the Help Text documentation changes.

Side task: Quick update to Slider documentation for clarification.

@Design-Kat @dmcalester @Cheshire89 please review.

### CODE REVIEW

**Review by Design-Kat — APPROVED** [review]

File: _content/patterns/forms-and-validation.md [hunk 1]
```diff
@@ -112,14 +113,22 @@ Tips for writing validation error messages in the voice of Astro:
...
+![Required Input - User must enter some text. If no text is entered, a warning icon and instructional text is shown.](/img/patterns/val-required-input.png 'Required Input - User must enter some text. If no text is entered, a warning icon and instructional text is shown.')
 
-![Numeric Range - input must be a number within a specific range. Non-numeric characters and out-of-range values are indicated with a warning icon and optional instructional text.](/img/patterns/val-numeric-range.png 'Numeric Range - input must be a number within a specific range. Non-numeric characters and out-of-range values are indicated with a warning icon and optional instructional text.')
+![Numeric Range - Input must be a number within a specific range. Non-numeric characters and out-of-range values are indicated with a warning icon and instructional text.](/img/patterns/val-numeric-range.png 'Numeric Range - Input must be a number within a specific range. Non-numeric characters and out-of-range values are indicated with a warning icon and instructional text.')
```
Comment (Cheshire89) [inline _content/patterns/forms-and-validation.md:129]: @amalyah  In html there is no such thing as numeric range  there is either numeric input or range input. Both however accept min and max attributes (range requires them) to set limits on acceptable values. Given that this paragraph is a bit miss leading

**Review by Cheshire89 — CHANGES_REQUESTED** [review]
refer to comments

File: _content/patterns/forms-and-validation.md (hunk 1 above)
Comment (amalyah) [reply _content/patterns/forms-and-validation.md:129]: FYI @Design-Kat

Comment (amalyah) [reply _content/patterns/forms-and-validation.md:129]: I only updated a bit of the wording in this statement and I'm not sure who originally wrote the rules, but I will try to disentangle this one for you.

**Review by Cheshire89 — APPROVED** [review]

**Review by dmcalester — APPROVED** [review]

**Pull request merged**